Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -16,128 +16,167 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <data>
-        <variable
-            name="email"
-            type="com.materialstudies.reply.data.Email" />
-    </data>
-
+  <data>
+    <variable
+        name="email"
+        type="com.materialstudies.reply.data.Email" />
+  </data>
+
+  <androidx.coordinatorlayout.widget.CoordinatorLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent">
+
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/email_app_bar_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingVertical="@dimen/grid_1"
+        android:paddingHorizontal="@dimen/grid_2"
+        android:fitsSystemWindows="true">
+      <com.google.android.material.appbar.MaterialToolbar
```

Comment:
Do you mind putting a return between each View block (e.g. [fragment_learn.xml ](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/layout/fragment_learn.xml)from Owl)? In general, I would try to follow the existing formatting that's in this project and `Owl`